setup.py

```diff
@@ -5,27 +5,42 @@
 import sys
 from pathlib import Path
 
-from setuptools import setup
-from setuptools.command.egg_info import egg_info
+from setuptools import Command, setup
+from setuptools.command.build import build
```

Comment:
linter complains about stubs not being installed, so I assume they exist and should be added/installed.

Reply:
I do not care to properly fix type hints in `setup.py`. `setuptools` isn't even installed in our venv normally so we need an entire different venv for this.